pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go

```diff
@@ -2614,3 +2616,34 @@ func validatePersistentState(field *k8sfield.Path, spec *v1.VirtualMachineInstan
 
 	return
 }
+
+func validateFileMemoryBacking(field *k8sfield.Path, spec *v1.VirtualMachineInstanceSpec) (causes []metav1.StatusCause) {
+	if !utils.IsFileMemoryBackedVmi(&v1.VirtualMachineInstance{Spec: *spec}) {
+		return
+	}
+
+	getField := func() string {
```

Comment:
You can just reuse the `field` variable.

Reply:
Maybe it's over-thinking, but the rational was to avoid the duplication of `field.Child("domain", "memory", "backed", "file").String()` in both `if` branches. I could just assign it to a variable once and re-use it, but that's unnecessary if no error occurred, therefore benefits performance.

But again, maybe it's over-thinking.